pom.xml

```diff
@@ -1276,6 +1275,26 @@
               </rules>
             </configuration>
           </execution>
+          <!-- Make sure the old ckeditor contrib extension is not used. -->
+          <execution>
+            <id>enforce-ckeditor-platform</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <bannedDependencies>
+                  <searchTransitive>true</searchTransitive>
+                  <message>
+                    Do not use the CKEditor extension from contrib. Is it now packaged in xwiki platform.
+                  </message>
+                  <excludes>
+                    <exclude>org.xwiki.contrib:application-ckeditor*:*</exclude>
```

Comment:
Why the `:*` ?

Reply:
My point is more that it does not seem needed, `org.xwiki.contrib:application-ckeditor*` should be enough, no ? We generally don't indicate the type there.